src/cdk/drag-drop/drop-list.ts

```diff
@@ -101,6 +101,10 @@ export class CdkDropList<T = any> implements OnInit, OnDestroy {
   @Output('cdkDropListExited')
   exited: EventEmitter<CdkDragExit<T>> = new EventEmitter<CdkDragExit<T>>();
 
+  /** Emits as the user is swapping items while dragging. */
```

Comment:
Could you say "actively dragging" just to make it more clear that it while the interaction is occurring?